app/classifier/tasks/survey/choice.spec.js

```diff
@@ -29,17 +29,31 @@ describe('Choice', function () {
   let wrapper;
   let answer;
 
-  describe('with single answer questions', function () {
-    beforeEach(function () {
-      wrapper = shallow(<Choice
-        translation={task}
-        task={task}
-        annotation={annotation}
-        annotationValue={annotationValue}
-        choiceID='ar'
-      />);
-      answer = wrapper.find('input[name="ho"][value="two"]');
+  beforeEach(function () {
+    wrapper = shallow(<Choice
+      translation={task}
+      task={task}
+      annotation={annotation}
+      annotationValue={annotationValue}
+      choiceID='ar'
+    />);
+    answer = wrapper.find('input[name="ho"][value="two"]');
+  });
+
+  describe('with confused with options', function () {
+    it('should render confusions', function () {
+      const confusions = wrapper.find('.survey-task-confusions-modal');
+      assert.equal(confusions.length, task.choices['ar'].confusionsOrder.length);
+    });
+    it('should render the confusions with appropriate labels', function () {
+      const confusions = wrapper.find('.survey-task-confusions-modal');
+      confusions.forEach(function (confusion, i) {
```

Comment:
Based on a comment from @eatyourgreens during today's standup, maybe this is a bad approach, should make more explicit like:
```
assert.equal(confusions.at(1).dive().find('.survey-task-choice-confusion').text(), task.choices[task.choices['ar'].confusionsOrder[i]].label)
```

Also, should I programmatically get the confusion label (like above), or just use `'Tortoise'`?

Reply:
Maybe test how many confusions exist before you iterate over them? You know how many are specified in the mock data.

Reply:
That makes sense, in which case I could combine these two tests into one test to 'render the confusions with appropriate labels'.